.gitignore

```
@@ -25,6 +25,9 @@
 .vagrant
 .capistrano
 
+# avoid checking in stray files
+.bak
```

I think this only blocks a file called `.bak`, not files suffixed as such.

We'd want this instead?

```suggestion
*.bak
```

---

Ty!